lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -11,14 +12,14 @@
 
 <%include file="/courseware/course_navigation.html" args="active_page='teams'" />
 
+<div class="team-tab-header"></div>
```

Comment:
I'd like to see this mirror the HTML in the UX reference:
https://github.com/edx/edx-platform/blob/frances/teams-ux/lms/templates/ux/reference/teams-base.html#L130

This would be:

    <div class="container">
        <div class="course-wrapper">
        <section class="course-content" id="course-content">
            <header class="page-header has-secondary">

This means that the header view can't look for a particular header class, because that is included in its own Underscore template. I think it would be best if the TeamsTabView created the header itself, and inserted it into the DOM in the correct location.